sql/mysql/Positive-Technologies/MySqlParser.g4

```diff
@@ -350,6 +350,7 @@ columnDefinition
 columnConstraint
     : nullNotnull                                                   #nullColumnConstraint
     | DEFAULT defaultValue                                          #defaultColumnConstraint
+    | (VISIBLE | INVISIBLE)                                         #invisibleColumnConstraint
```

Comment:
I suggest using the first level alternatives if possible (it may improve performance):

```suggestion
    | VISIBLE                                             #invisibleColumnConstraint
    | INVISIBLE                                           #invisibleColumnConstraint
```

Reply:
Updated as your suggestion